modules/configuring-the-distributed-workloads-components.adoc

```diff
@@ -0,0 +1,194 @@
+:_module-type: PROCEDURE
+
+[id="configuring-the-distributed-workloads-components_{context}"]
+= Configuring the distributed workloads components
+
+[role='_abstract']
+To configure the distributed workloads feature for your data scientists to use in {productname-short}, you must enable several components.
+
+.Prerequisites
+ifdef::upstream,self-managed[]
+* You have logged in to {openshift-platform} with the `cluster-admin` role.
+endif::[]
+ifdef::cloud-service[]
+* You have logged in to OpenShift with the `cluster-admin` role.
+endif::[]
+
+* You have access to the data science cluster.
```

Comment:
combine first two bullet items into one:
 You have access to the cluster where ODH/RHOAI is installed.

Reply:
I think we need to standardize this wording across all modules, I've added an item to our team agenda to discuss.